cmd/spire-server/cli/entry/create.go

```diff
@@ -263,3 +248,37 @@ func getParentID(config *createCommand, td string) (*types.SPIFFEID, error) {
 	}
 	return idStringToProto(config.parentID)
 }
+
+func prettyPrintCreate(env *commoncli.Env, results ...interface{}) error {
+	var succeeded, failed []*entryv1.BatchCreateEntryResponse_Result
+	createResp, ok := results[0].(*entryv1.BatchCreateEntryResponse)
+	if !ok {
+		return errors.New("internal error: cli printer; please report this bug")
```

Comment:
I think that we may have this error in a constant in the `cliprinter` package since it's a generic error for the printer.

Reply:
Awesome suggestion @amartinezfayo! going to update the branch.